indexing-service/src/test/java/org/apache/druid/indexing/overlord/sampler/FirehoseSamplerTest.java

```diff
@@ -814,9 +814,7 @@ private ParseSpec getParseSpec(TimestampSpec timestampSpec, DimensionsSpec dimen
   private String getUnparseableTimestampString()
   {
     return ParserType.STR_CSV.equals(parserType)
-           ? (USE_DEFAULT_VALUE_FOR_NULL
-              ? "Unparseable timestamp found! Event: {t=bad_timestamp, dim1=foo, dim2=null, met1=6}"
-              : "Unparseable timestamp found! Event: {t=bad_timestamp, dim1=foo, dim2=, met1=6}")
+           ? "Unparseable timestamp found! Event: {t=bad_timestamp, dim1=foo, dim2=null, met1=6}"
```

Comment:
@nishantmonu51 Please let me know if you think there is a better way to do this. 
With the CSVParser fix in this PR, the way to specify empty values for csv files would be `''"`
One of the test data samples is `bad_timestamp,foo,,6` and dim2 would be null regardless of whether null handling is enabled or not.

Reply:
I think this should be ok. @nishantmonu51 do you have something in your mind?